atlp-rwanda/atlp-devpulse-bn
# Ft schedule technical interview

### PR Description

This PR allows the admin to invite a trainee applicant to the technical interview

### Description of tasks that were expected to be completed

As a trainee applicant, I will receive an email with a meeting link for the interview invitation.
As an admin, I should be able to create, view, and update the status of the interview

### How can this be tested?
- Clone repo
- Checkout branch [here](ft-schedule-technical-interview)
- Make sure that you set the environment variables referring to `.env.example`
- Run `npm install`, then `npm run dev` to check if the app is running successfully
- Use Apollo server and interact with doc resolvers


### Track PR (issue number & link)
Issue: [#208](https://github.com/atlp-rwanda/atlp-devpulse-fn/issues/208)

### Screenshots (If appropriate especially on the frontend)
N/A
